### DISCUSSION TITLE
While real time insert existing in a cluster,how to change external pulsar

### DISCUSSION THREAD

**Discussion by bingbb47** [milvus-io/milvus#26385] [Q&A and General discussion] [answered]
If I directly modify the configuration file and restart the pods, will it cause any collection to be damaged and unable to load.

**Comment by yhmo** [comment]
If you stopped the pulsar server, insert/delete requests will be blocked until the pulsar server is back to work. Theoretically no damage risk.

**Reply by bingbb47** [reply]
will search be blocked? its the most critical, hhh

**Reply by yhmo** [reply, marked as answer]
Search with consistency_level=Strong/Session/Bounded will be blocked since the query node needs to check the time tick queue which is stored in pulsar.

Search with consistency_level=Eventually will not be blocked since the query node doesn't check time tick.

**Reply by bingbb47** [reply]
thank you! its helpful
